x-pack/test/functional/apps/infra/index.ts

```diff
@@ -15,11 +15,11 @@ export default ({ loadTestFile }: FtrProviderContext) => {
 
     describe('Metrics UI', function () {
       loadTestFile(require.resolve('./home_page'));
-      loadTestFile(require.resolve('./metrics_source_configuration'));
       loadTestFile(require.resolve('./metrics_anomalies'));
       loadTestFile(require.resolve('./metrics_explorer'));
       loadTestFile(require.resolve('./node_details'));
       loadTestFile(require.resolve('./hosts_view'));
+      loadTestFile(require.resolve('./metrics_source_configuration'));
```

Comment:
>[!NOTE]
> This test is potentially dangerous because it changes the index pattern, which could affect al subsequent tests if it fails to return the data to the original state.